_inc/client/traffic/shortlinks.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { translate as __ } from 'i18n-calypso';
+/**
+ * Internal dependencies
+ */
+import { withModuleSettingsFormHelpers } from 'components/module-settings/with-module-settings-form-helpers';
+import { getModule } from 'state/modules';
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+import { ModuleToggle } from 'components/module-toggle';
+
+class Shortlinks extends Component {
+	render() {
+		const isLinked = this.props.isLinked;
```

Comment:
I don't know that `isLinked` is available as a prop here. I believe you'll need to add it to `_inc/client/traffic/index.jsx` first.

Reply:
Fixed.